dom/events/enter/enter-test.js

```diff
@@ -14,6 +14,9 @@ function makeKeyboardEvent() {
 		event = new KeyboardEvent("keyup", {
 			key: "Enter"
 		});
+		Object.defineProperty(event, "type", {
```

Comment:
jquery does an `event.hasOwnProperty("type")`. If that fails, it does `event.indexOf(...)` which throws an error with KeyboardEvents. Anyway, I reverted this change and moved it into can-jquery: https://github.com/canjs/can-jquery/pull/86/files.